src/validateRegisterForm.test.js

```diff
@@ -19,12 +19,82 @@ describe(`Function 'validateRegisterForm':`, () => {
     expect(isValid.message).toBe('Email and password are valid.');
   });
 
-  it(`should return error for valid email and password without number`, () => {
+  it(`should return error for valid email and password
+   without number`, () => {
     const invalidPassword = validateRegisterForm('test@mail.com', 'P@ssword');
 
     expect(invalidPassword.code).toBe(422);
     expect(invalidPassword.message).toBe('Password is invalid.');
   });
```

Comment:
Not all combinations of email and password were covered, please revise requirements and cover by tests all possible combinations (remember about test design techniques)